vortexasdk/endpoints/voyages_search_enriched.py

```diff
@@ -5,6 +5,7 @@
 """
 from datetime import datetime
 from typing import Any, Dict, List, Union
+from typing_extensions import Literal
```

Comment:
This change made me realise we're being too lenient with our API param type definitions. For example, `vessel_risk_level: Union[str, List[str]] = None,` ->  we say this may be any random string, but this is not quite right

Could you please create a follow up ticket to:
- check all endpoints for params which are not strictly typed (mainly strings, but there may be others)
- introduce a similar Literal type wherever appropriate

You can add this ticket to one of the debt work packages, someone will pick it up at a later time